content/tutorials/gac/index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Gatling as Code"
+description: "Setup tutorial"
+lead: "Set up your project for automatic configuration of your simulation in Gatling Enterprise."
+date: 2024-01-31T18:30:00+02:00
+lastmod: 2024-02-01T11:00:00+00:00
+weight: 2090700
+private: true
+---
+
+{{<alert tip>}}
+If you need a package to test this, you can use the following one: [`gatling-maven-plugin-demo-java`](https://github.com/gatling/gatling-maven-plugin-demo-java)
+{{</alert>}}
+# Configuration
+
+## 1. Edit your `pom.xml` file
+
+ * In `project.properties`, make sure you use the following version:
+   ```xml
+   <gatling-maven-plugin.version>4.8.0-M1</gatling-maven-plugin.version>
+   ```
+   * In `project.build.plugins.plugin.configuration` (corresponding to gatling-maven-plugin), fill in your apiToken:
+     ```xml
+     <project>
+       <build>
+         <plugins>
+          <plugin>
+            <groupId>io.gatling</groupId>
+            <artifactId>gatling-maven-plugin</artifactId>
+            <version>${gatling-maven-plugin.version}</version>
+            <configuration>
+              <apiToken>XXXXX</apiToken> <!-- Put your own API Token -->
+            </configuration>
+          </plugin>
+         </plugins>
+       </build>
+     </project>
+     ```
+     
+     {{<alert tip>}}
+Your [API token]({{<ref "../../reference/admin/api_tokens">}}) must have the **Configure** role.
+     {{</alert>}}
+
+## 2. Create your Gatling configuration
+
+Add a new directory at the root of your project: `.gatling` (sibling of your `pom.xml` file).
+
+Create a new file `.gatling/package.conf`.
+
+```console
+.
+├── .gatling/
+│   └── package.conf
+├── src/
+│   ├── main/
+│   └── test/
+└── pom.xml
+```
+
+This file is in [`HOCON` format (Human-Optimized Config Object Notation)](https://github.com/lightbend/config/blob/main/HOCON.md), which means you can also write `JSON` if you prefer.
+
+```bash
+gatling.enterprise {
+  # The name of the package (Mandatory)
+  name = "My package name"
+  # The team UUID (Mandatory)
+  # You can copy it from Organization / Teams / More / Copy ID
+  team = "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
+
+  # Simulations based on the package (Optional)
+  # List of objects [{...}, {...}, {...}]
+  simulations = [{
+    # The name of the simulation (Mandatory)
+    name = "My simulation name"
+
+    # The FQDN of the class that extends Simulation (Mandatory)
+    classname = "com.example.MySimulation"
+
+    ## Using or not dedicated IPs (optional)
+    ## Only available if dedicated IPs are configured in your organization
+    # useDedicatedIps = false
+
+    # Locations configuration (mandatory)

```

Comment:
Is it not currently compatible with Private Locations?

Reply:
Not yet

Reply:
Why not? Sorry, I don't understand.

Reply:
1. Not in scope
2. We are looking in the public location for the name to find the ID. (So we'll have to change that)

Reply:
> Not in scope

Was it mentioned it was not is scope? I missed that.
JustEats isn't on Private Locations (yet?).
But Disney is. IMO, This must be implemented in a follow up soon.

> We are looking in the public location for the name to find the ID. (So we'll have to change that)

Not very complicated to add.

Reply:
> Not very complicated to add.

Exactly.

We will add this ticket in the follow-up epic.

Reply:
Changed:

 * Removed completely the api token configuration in XML.
 * Put relevant links right before executing the command.